ethexe/runtime/common/src/lib.rs

```diff
@@ -151,8 +152,8 @@ where
         gas_multiplier: GasMultiplier::one(),
         costs: Default::default(),
         max_pages: MAX_WASM_PAGES_AMOUNT.into(),
-        outgoing_limit: 1024,
-        outgoing_bytes_limit: 64 * 1024 * 1024,
+        outgoing_limit: OUTGOING_LIMIT,
```

Comment:
wdyt if we remove this constants from configs at all? or provide some default constructor with them not to miss anything, @grishasobol ?